modules/apps/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/edit_permissions.jsp

```diff
@@ -38,7 +38,7 @@ PortletConfigurationPermissionPropagation portletConfigurationPermissionPropagat
 			selectable="<%= false %>"
 		/>
 
-		<c:if test="<%= (portletConfigurationPermissionPropagation != null) && portletConfigurationPermissionPropagation.isAvailable(renderRequest) %>">
+		<c:if test='<%= FeatureFlagManagerUtil.isEnabled("LPS-154515") && (portletConfigurationPermissionPropagation != null) && portletConfigurationPermissionPropagation.isAvailable(renderRequest) %>'>
```

Comment:
How about returning null [here](https://github.com/tototrinh/liferay-portal/blob/LPS-154515/modules/apps/portlet-configuration/portlet-configuration-web/src/main/java/com/liferay/portlet/configuration/web/internal/display/context/PortletConfigurationPermissionsDisplayContext.java#L311-L316) if the feature is not enabled?